api/src/pcapi/core/educational/models.py

```diff
@@ -768,7 +768,9 @@ def isReleased(self) -> bool:
             and self.venue.managingOfferer.isValidated
         )
 
-    is_eligible_for_search = isReleased
+    @property
+    def is_eligible_for_search(self) -> bool:
+        return bool(self.isReleased and self.venue.postalCode)
```

Comment:
Pourquoi `self.venue.postalCode` et pas `not self.venue.isVirtual` ? (c’est une vraie question)

Reply:
Bien vu, j'ai fais au plus simple vu que c'était l'absence d'une adresse qui posait problème mais le `isVirtual` semble correct.